sql/expression/function/string_test.go

```diff
@@ -67,16 +67,16 @@ func TestUnhexFunc(t *testing.T) {
 	f := sql.Function1{Name: "unhex", Fn: NewUnhex}
 	tf := NewTestFactory(f.Fn)
 	tf.AddSucceeding(nil, nil)
-	tf.AddSucceeding("MySQL", "4D7953514C")
-	tf.AddSucceeding("\x01#Eg\x89\xab\xcd\xef", "0123456789abcdef")
-	tf.AddSucceeding("\x8f", "8F")
-	tf.AddSucceeding("\x8f", "8f")
-	tf.AddSucceeding("\x0b", "B")
+	tf.AddSucceeding([]byte{0x4d, 0x79, 0x53, 0x51, 0x4c}, "4D7953514C")
```

Comment:
Probably easiest to just []byte() on the string of hex value, rather than using the ascii code points here